src/components/Attendance/Table.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import "./Attendace.css";
+const URL = "https://jsonplaceholder.typicode.com/users";
+
+const Table = (props) => {
+  const [employees, setEmployees] = useState([]);
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const getData = async () => {
+    const response = await axios.get(URL);
```

Comment:
Put all of your fetch logic in one file maybe in an api or services folder and import it like we did in class. It should not be in the component.